orbit/pkg/packaging/macos_templates.go

```diff
@@ -26,7 +26,11 @@ var macosDistributionTemplate = template.Must(template.New("").Option("missingke
     <choice id="choiceBase" title="Fleet osquery" enabled="false" selected="true" description="Standard installation for Fleet osquery.">
         <pkg-ref id="{{.Identifier}}.base.pkg"/>
     </choice>
-    <pkg-ref id="{{.Identifier}}.base.pkg" version="{{.Version}}" auth="root">#base.pkg</pkg-ref>
+    <pkg-ref id="{{.Identifier}}.base.pkg" version="{{.Version}}" auth="root">
+      <bundle-version>
+        <bundle id="{{.Identifier}}" path="{{.Identifier}}.base.pkg" />
+      </bundle-version>
+    </pkg-ref>
```

Comment:
the tools we use were having problems finding the bundle ID, this was updated based on what the docs describe [here](https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/DistributionDefinitionRef/Chapters/Distribution_XML_Ref.html).

@lucasmrod from the docs seems like an string (the previous value) is not a valid value, do you know if by any chance `#base.pkg` has special meaning, or it was just wrong?

Reply:
> Good question. It does look invalid, but maybe it's used by bomutils?

sounds good, let's see what Zack says.

> (I have no idea about this, just asking) Should be using {{.Version}}?

I'm no expert either, but I believe `.Identifier` is more appropriate, this is the documentation for the `bundle.id` attribute:

> Required. The identifier of the bundle—that is, the value of CFBundleIdentifier in the bundle’s Info.plist file.

Reply:
We don't use `productbuild` since that's a macos only tool, so I can see how `bundle-version` might actually need to be specified.

I don't have any memory of `#base.pkg` having any special meaning, probably a typo.